node/node.go

```diff
@@ -856,7 +856,7 @@ func (node *Node) StartMining(ctx context.Context) error {
 						continue
 					}
 
-					node.StorageMiner.OnCommitmentAddedToChain(val, nil)
+					node.StorageMiner.OnCommitmentAddedToChain(val, msgCid, nil)
```

Comment:
It's not on the chain yet, so this call is misleading. Could you rename it to `OnCommitmentSent` or similar. I'd expect an `OnCommitmentAddedToChain` to be callback in response to observing it mined in a block or similar.

Reply:
@anorth - You make a good point. 

@acruikshank - You can call `MessageWait` after calling `MessageSend`, and call `OnCommitmentAddedToChain` in the function passed to `MessageWait`. That would make these names not-a-lie.

Reply:
Good call.

Reply:
@laser I'd rather just make the names right. At least right now. The client is going to have to MessageWait anyway to compensate for chain sync timing.